content/_index.md

```diff
@@ -27,9 +27,12 @@ a.linkhighlight:hover:after, a.linkhighlight:focus:after {
 
 .centerimage {
   height: 50%;
+  width:100%;
   display: block;
   text-align:center;
   display: flex;
+  margin-top:90px;
+  margin-bottom:110px;
```

Comment:
I think the margins are too big. Can we reduce them?

Reply:
Sure @pkosiec ! "what can botkube do?" text right?

Reply:
What do you mean? 🤔  The headline should be positioned as it is now (right window on the screenshot) 🙂

Reply:
I was just confirming the text !
yeahhh godit!

Reply:
Could you pls mention the dimensions where you find this issue?

Reply:
Unfortunately I didn't note them, but you can play around with window dimensions or Chrome Developer Tools  to see how the text floats 🙂

Reply:
Cool. will check that

Reply:
Hi @pkosiec .I've created a PR. Kindly check

Reply:
Kindly please update this PR instead of creating a new one, as it is about fixing this PR 🙂 Let's have our discussion in one place and track the changes here. Thank you!

Reply:
My bad!
updated here itself!